scripts/api-ja.js

```diff
@@ -0,0 +1,236 @@
+// const fetch = require('node-fetch');
```

Comment:
Can we remove this file if we don't need it?

Reply:
Yes, it can now be deleted. It can also be removed from the current build script.

Reply:
Sorry, I should have asked this originally, but can you clarify why we don't need this script anymore? In `main` it looks like we still use it: https://github.com/ionic-team/ionic-docs/blob/e9c3e62bb83a13392145d5f2d836b4925e6a8540/scripts/i18n.sh#L8

Reply:
Yes, remove from main branch will not effect. Files no longer in use now.

However, this issue needs to be discussed: https://github.com/ionic-team/ionic-docs/issues/3096

Reply:
Thanks for the clarification!